app/src/main/res/layout/fragment_fever_temperature_spot.xml

```diff
@@ -0,0 +1,174 @@
+                android:background="@drawable/circle_button_background"
+                android:onClick="@{() -> vm.onClickMouth()}">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="18dp"
+                    android:layout_marginRight="18dp"
+                    android:gravity="center"
+                    android:letterSpacing="0.01"
+                    android:lineSpacingExtra="1sp"
+                    android:text="@string/symptom_report_fever_temperature_spot_mouth"
+                    android:textColor="@color/button_text_color"
+                    android:textSize="24sp"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"/>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="126dp"
+                android:layout_height="126dp"
+                android:background="@drawable/circle_button_background"
+                android:onClick="@{() -> vm.onClickEar()}">
+
+                <TextView
+                    android:layout_width="0dp"
+                    android:layout_height="wrap_content"
+                    android:layout_marginLeft="18dp"
+                    android:layout_marginRight="18dp"
+                    android:gravity="center"
+                    android:letterSpacing="0.01"
+                    android:lineSpacingExtra="1sp"
+                    android:text="@string/symptom_report_fever_temperature_spot_ear"
+                    android:textColor="@color/button_text_color"
+                    android:textSize="24sp"
+                    android:textStyle="bold"
+                    app:layout_constraintEnd_toEndOf="parent"
+                    app:layout_constraintStart_toStartOf="parent"
+                    app:layout_constraintTop_toTopOf="parent"
+                    app:layout_constraintBottom_toBottomOf="parent"/>
+            </androidx.constraintlayout.widget.ConstraintLayout>
+        </LinearLayout>
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="horizontal"
+                android:layout_marginTop="38dp"
+                android:gravity="center">
+
+                <androidx.constraintlayout.widget.ConstraintLayout
+                    android:layout_width="126dp"
+                    android:layout_height="126dp"
+                    android:layout_marginEnd="46dp"
+                    android:background="@drawable/circle_button_background"
```

Comment:
I'd prefer if these buttons are in `RecyclerView`, but let's keep it like this for now. We can refactor the UI later. But at least create reusable style(s) / dimensions, so we don't have to edit every button if we want to e.g. change the size or the background drawable.

Reply:
Moved a lot of the dimensions and style elements into styles for the buttons, also did the same for elements reused across the pages such as the headline and the toolbars.